softdev.bib

```diff
@@ -2590,7 +2590,7 @@ @article{woodruff19chericoncentrate
       Roe, Michael and Neumann, Peter G. and Watson, Robert N. M. and
       Moore, Simon W.",
     title = "{CHERI} {Concentrate}: Practical Compressed Capabilities",
-    journal = {IEEE Transactions on Computers},
+    journal = {TCOM},
```

Comment:
When I search "IEEE TCOM", I get suggested "Transactions on Communications". Can't find an appropriate acronym for "Transactions on Computers". If that's not an issue, we can go ahead with this acronym as is.

Reply:
Oops! Re-expanded in 08ff05d.